packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/quanticRefineModalContent.js

```diff
@@ -240,6 +266,14 @@ export default class QuanticRefineModalContent extends LightningElement {
     return facetData;
   }
 
+  /**
+   * Returns sort option data.
+   * @returns {Array<SortOptionObject>}
+   */
+  get sortOptions() {
+    return this.sortData || [];
```

Comment:
`this.sortData` will automatically be `undefined` until you set a value to it. so if you already are doing `|| []` then there's no need to keep a variable for `this.sortInitialized`

In your template you can only keep the `for:each sortOptions` and it'll receive an empty array until initialized. So it won't render anything.